src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1455,12 +1455,21 @@ public ChangeLogParser createChangeLogParser() {
         private String globalConfigEmail;
         private boolean createAccountBasedOnEmail;
 //        private GitClientType defaultClientType = GitClientType.GITCLI;
+        private boolean truncateSummary;
```

Comment:
This will need to default to `true`, since the current behavior is to truncate the summary.  

I believe boolean values default to false, so this will have the wrong initial value and will cause incompatible behavior when users upgrade to the plugin release which includes this change.

Reply:
as Jenkins will always use the two param constructor, the value of this will be coming from the configuration. There is a default value for a system failure in the method that retrieves that.
Is initialised in the constructor.
Should br good like that